### DISCUSSION TITLE
Simplify SQL Builder and Model types generation

### DISCUSSION THREAD

**Discussion by kcmvp** [go-jet/jet#313] [General]
**Is your feature request related to a problem? Please describe.**
According to the document, the generator need a running database to generate the model& builder, in the daily development it’s not so convenient.
**Describe the solution you'd 
May be we can use a sql parser to parse the schema file to generate the stuffs, for those stuffs the schema sql file is enough.

**Comment by houtn11** [comment]
This has been proposed before. Check this discussions: https://github.com/go-jet/jet/discussions/150, https://github.com/go-jet/jet/discussions/165
